examples/showcase-all-resources.tf

```diff
@@ -0,0 +1,105 @@
+# App
+
+resource "ably-app" "app1" {
+  status                 = "enabled"
+  tlsOnly                = "false"
+  fcmKey                 = "AABBQ1KyxCE:APA91bCCYs7r_Q-sqW8HMP_hV4t3vMYx...cJ8344-MhGWODZEuAmg_J4MUJcVQEyDn...I"
+  apnsCertificate        = "-----BEGIN CERTIFICATE-----MIIFaDCC...EXAMPLE...3Dc=-----END CERTIFICATE-----"
+  apnsPrivateKey         = "-----BEGIN PRIVATE KEY-----ABCFaDCC...EXAMPLE...3Dc=-----END PRIVATE KEY-----"
+  apnsUseSandboxEndpoint = false
+}
+
+# Keys
+
+resource "ably-api-key" "api-key-1" {
+  app  = ably-app.app1.app_id
```

Comment:
change this one to app_id

Reply:
Good point! Done in ad51ea48e0390527496dd9e01c88f69c314e737b